roles/network_plugin/calico/tasks/install.yml

```diff
@@ -6,6 +6,27 @@
     mode: 0755
     remote_src: yes
 
+- name: Calico | Check if host has NetworkManager
+  stat:
+    path: "/etc/NetworkManager/conf.d"
```

Comment:
why are you both checking if it exists and the creates the exact same path ? 
shoudn't be :
`path:  "/etc/NetworkManager/"`
instead of:
`path:  "/etc/NetworkManager/conf.d"`
?

Reply:
oh I see what you mean. I think I should change the task to checking if the service is running or enabled actually

Reply:
Yes